### ISSUE TITLE
bug: Mobile notifications just don't work

### ISSUE THREAD

**Issue by crxssed7** [revoltchat/revite#871]
### What happened?

Notifications on mobile (both Android and iOS) only work when the app is open. I suspect this is due to it being a webapp. However since this is a chat app I believe push notifications (that are sent regardless of whether the app is open or not) should be a vital feature of the mobile apps. I don't know how this could be done, but there could be an alternate solution such as email notifications.

### Branch

Production (app.revolt.chat)

### Commit hash

e17e556

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### Desktop

- [ ] Yes, this bug is specific to Revolt Desktop and is *not* an issue with Revolt Desktop itself.

### PWA

- [X] Yes, this bug is specific to the PWA.

**Comment by Gyztor** [comment]
I have noticed this too, i seem to be unable to enable the setting for push notifications on the android app or safari web interface. But seems to work fine on desktop browsers or applications.

**Comment by JustChamp** [comment]
Still not working properly, at least for android, I only get notifications when I open the app (If I even get them then)

**Comment by Typednull** [comment]
Yeah, even now i don't get any mobile notifications. Hope this can be fixed somehow

**Comment by orklepee** [comment]
Issue persists for me as of today.

**Comment by ilmaisin** [comment]
On Firefox for Android, it is impossible to check the "Push" notifications option: the check mark appears on the check box, but the box stays gray, and the mark vanishes when I tap somewhere else.

**Comment by erichmond33** [comment]
Just spent many hours trying to fix this. Still can't get it working. plz help.

**Comment by ScaleShiftX** [comment]
This issue is still present as of today.

Devices where this issue has been observed:
- Desktop app, Windows 11 desktop, 25H2 (OS Build 26200.7462), my desktop (have gotten *one* notif so far, but otherwise none)
- stoat.chat Website, Firefox 147.0.3 (64-bit), my desktop
- Android 16, BP4A.260105.004.E1, on Pixel 8, my phone
- Android 14, RMX3472_14.0.0.501(EX01), on Realme 9, my friend AntRox's phone
- Android 16, BP2A.250605.031.A3.F731USQS6FZA1, on Flip 5, my friend Sage's phone

For each Android phone we verified that notifications for Stoat were turned on in system settings.

Devices where I HAVE gotten notifications:
- Desktop app, Windows 11 desktop, 25H2 (OS Build 26200.7462), my desktop (have gotten *one* notif so far, but otherwise none)

Tested DMs only.

Since the start, I got no notifs on Windows, until one recently did one come through. But other than that one, none have come through.
